src/python_testing/TC_DEMTestBase.py

```
@@ -261,3 +363,9 @@ async def send_test_event_trigger_forecast(self):
 
     async def send_test_event_trigger_forecast_clear(self):
         await self.send_test_event_triggers(eventTrigger=0x0098000000000010)
+
+    async def send_test_event_trigger_power_range_adjustment(self):
+        await self.send_test_event_triggers(eventTrigger=0x0098000000000011)
```

Pylint (4.0.6)

[error] Instance of 'DEMTestBase' has no 'send_test_event_triggers' member

(E1101)